src/ui.rs

```diff
@@ -88,36 +97,62 @@ pub struct UiCell<'a> {
 /// buffer. Otherwise if we don't draw into each buffer, we will probably be subject to flickering.
 pub const SAFE_REDRAW_COUNT: u8 = 3;
 
+impl UiBuilder {
+    /// Create **Ui** builder with defaults.
+    ///
+    /// - theme: Theme::default()
+    /// - maybe_widgets_capacity: None
+    pub fn new() -> Self {
+        UiBuilder {
+            maybe_theme: None,
+            maybe_widgets_capacity: None
+        }
+    }
 
-impl Ui {
+    /// Sets the theme of built **Ui**.
+    ///
+    /// This method moves the current builder,
+    /// unlike [`set_theme()`](#method.set_theme),
```

Comment:
There is no longer any `set_theme` method, perhaps these doc comments should be updated?